src/ui-presets/playback.js

```diff
@@ -36,7 +39,12 @@ import {ShareControl} from '../components/share';
  */
 export function playbackUI(props: any): React$Element<any> {
   return (
-    <div className={style.playbackGuiWWrapper}>
+    <Container
+      className={style.playbackGuiWWrapper}
+      name={'VideoOverlay'}
```

Comment:
we need to have a fixed container convention.
This is what we currently have
```html
<Main>
  <PlayerGui> - this is not a container for now in your design
    <TopBarLeftControls/>
    <TopBarRightControls/>
    <BottomBarLeftControls/>
    <BottomBarRightControls/>
  </PlayerGui>
</Main>
```

Seems we are missing some layout semantics to allow embedding UI in the staging area, e.g. on top of video and between bars.
So a full container layout might look like:
```html
<Main>
  <PlayerGui> <!-- this is not a container for now in your design-->
    <StagingContainer/> <!-- temporary name :-)-->
    <TopBarLeftControls/>
    <TopBarRightControls/>
    <BottomBarLeftControls/>
    <BottomBarRightControls/>
  </PlayerGui>
</Main>
```
Staging are can be an adjustable container that works in conjunction with the bars and can fit itself to the bars state, so in hover state the PlayerGui can tell the inner containers to adjust their display or something like this.